cicecore/cicedyn/general/ice_init.F90

```diff
@@ -322,7 +322,7 @@ subroutine input_data
       histfreq(4) = 'm'      ! output frequency option for different streams
       histfreq(5) = 'y'      ! output frequency option for different streams
       histfreq_n(:) = 1      ! output frequency
-      histfreq_base = 'zero' ! output frequency reference date
+      histfreq_base(:) = 'zero' ! output frequency reference date
```

Comment:
@DeniseWorthen is it possible we can extend this to allow random output time?

Reply:
I haven't looked into the various ways this feature can work yet, but that is why I want to go ahead and get it in so we can find out what it can do.

Reply:
Any reason they are trying to define allocation here and not in the variable definition?

Reply:
This is actually just in the namelist initialization to provide values which will be used if you don't provide an alternate value in the namelist.